src/stepfunctions/steps/sagemaker.py

```diff
@@ -24,7 +26,7 @@
 from sagemaker.model_monitor import DataCaptureConfig
 
 SAGEMAKER_SERVICE_NAME = "sagemaker"
-
+logger = logging.getLogger('stepfunctions.sagemaker')
```

Comment:
Try to pay attention to when you have unused variables. Your IDE should tell you this. Since you removed the logging in your last commit, this is unused now.

However, https://github.com/aws/aws-step-functions-data-science-sdk-python/pull/144 also added logging so you just need to resolve the merge conflict after merging from main.

https://github.com/aws/aws-step-functions-data-science-sdk-python/blob/main/src/stepfunctions/steps/sagemaker.py#L28

Reply:
Agreed!
The logger was removed in the latest commit